threads/src/main/java/org/jboss/as/threads/ManagedJBossThreadPoolExecutorService.java

```diff
@@ -4,101 +4,12 @@
  */
 package org.jboss.as.threads;
 
-import org.jboss.threads.EventListener;
-import org.jboss.threads.JBossThreadPoolExecutor;
-
 /**
  *
  * @author Alexey Loubyansky
+ *
+ * @deprecated Use {@link ManagedQueueExecutorService}
  */
-public class ManagedJBossThreadPoolExecutorService extends ManagedExecutorService  {
-
-    private final JBossThreadPoolExecutor executor;
-
-    public ManagedJBossThreadPoolExecutorService(JBossThreadPoolExecutor executor) {
-        super(executor);
-        this.executor = executor;
-    }
-
-    @Override
-    void internalShutdown() {
-        executor.shutdown();
-    }
-
-    public int getCoreThreads() {
-        return executor.getCoreThreads();
-    }
-
-    // Package protected for subsys write-attribute handlers
-    void setCoreThreads(int coreThreads) {
-        executor.setCoreThreads(coreThreads);
-    }
-
-    public boolean isAllowCoreTimeout() {
-        return executor.isAllowCoreThreadTimeout();
-    }
-
-    void setAllowCoreTimeout(boolean allowCoreTimeout) {
-        executor.setAllowCoreThreadTimeout(allowCoreTimeout);
-    }
-
-    public boolean isBlocking() {
-        return executor.isBlocking();
-    }
-
-    void setBlocking(boolean blocking) {
-        executor.setBlocking(blocking);
-    }
-
-    public int getMaxThreads() {
-        return executor.getMaxThreads();
-    }
-
-    void setMaxThreads(int maxThreads) {
-        executor.setMaxThreads(maxThreads);
-    }
-
-    public long getKeepAlive() {
-        return executor.getKeepAliveTime();
-    }
-
-    void setKeepAlive(TimeSpec keepAlive) {
-        executor.setKeepAliveTime(keepAlive.getDuration(), keepAlive.getUnit());
-    }
-
-    public int getRejectedCount() {
-        return executor.getRejectedCount();
-    }
-
-    public long getTaskCount() {
-        return executor.getTaskCount();
-    }
-
-    public int getLargestThreadCount() {
-        return executor.getLargestThreadCount();
-    }
-
-    public int getLargestPoolSize() {
-        return executor.getLargestPoolSize();
-    }
-
-    public int getCurrentThreadCount() {
-        return executor.getCurrentThreadCount();
-    }
-
-    public long getCompletedTaskCount() {
-        return executor.getCompletedTaskCount();
-    }
-
-    public int getActiveCount() {
-        return executor.getActiveCount();
-    }
-
-    public int getQueueSize() {
-        return executor.getQueueSize();
-    }
-
-    <A> void addShutdownListener(final EventListener<A> shutdownListener, final A attachment) {
-        executor.addShutdownListener(shutdownListener, attachment);
-    }
+@Deprecated(forRemoval = true)
```

Comment:
Ideally add a since for each deprecated so we now easier when it was deprecated

Reply:
Thanks, @yersan -- I've added that on this class and ManagedQueuelessExecutorService.